dbld/builddeps

```diff
@@ -187,9 +187,18 @@ function install_pip_packages {
             ;;
     esac
     for python_executable in ${python_executables}; do
-        $python_executable -m pip install --upgrade pip
-        $python_executable -m pip install --upgrade setuptools
-        filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs $python_executable -m pip install --ignore-installed -U
+        case "${OS_PLATFORM}" in
+            centos-6)
+                pip install --upgrade pip==9.0.3
+                pip install --upgrade setuptools
+                filter_packages_by_platform $DBLD_DIR/pip_packages.manifest | xargs pip install --ignore-installed -U
+                ;;
+            *)
+                $python_executable -m pip install --upgrade pip
+                $python_executable -m pip install --upgrade setuptools
```

Comment:
What if we had $pip_command instead $python_executable, so we can have: python2 -m pip, python3 -m pip, python -m pip, pip?

The only challenge would be the devshell, which would have multiple, but space won't be as a good delimiter, as it is now.

After that only the version forcing would be the difference, to which I don't have an idea yet.